# Automatically Generated in 2017-06-12 22:33:47.453127
# MESSAGE: CONFIG_PARAM_INT
# Description:Config parameter information for int param
uint8 ID = 197
uint8 sysid
uint8 compid
uint8 config_id
uint8 param_id
char[30] name
int32 value
int32 default_value
int32 min
int32 max
